src/Visualization/Python/GenerateXdmf.py

```diff
@@ -21,13 +21,20 @@ def generate_xdmf(file_prefix, output, subfile_name, start_time, stop_time,
                         "python 2, which is deprecated. GenerateXdmf.py might "
                         "hang or run very slowly using python 2. Please use "
                         "python 3 instead.")
+    if subfile_name is None:
+        raise ValueError("The --subfile-name option must be specified.")
+
     h5files = [(h5py.File(filename, 'r'), filename)
                for filename in glob.glob(file_prefix + "*.h5")]
 
     assert len(h5files) > 0, "No H5 files with prefix '{}' found.".format(
         file_prefix)
 
     element_data = h5files[0][0].get(subfile_name + '.vol')
+    if element_data is None:
+        raise ValueError("Could not open subfile name ",
+                         str(subfile_name + '.vol. Available options are:'),
+                         str(h5files[0][0].keys()))
```

Comment:
I think this prints an extra space after "name" or misses one after "are:". Just use proper string formatting instead of passing 3 strings to the constructor:

```suggestion
        raise ValueError(("Could not open subfile name '{}.vol'. Available "
                          "subfiles: {}").format(subfile_name,
                                                 h5files[0][0].keys()))
```

Note that you can break a string over multiple lines by enclosing it in parens, like above. Unfortunately [yapf doesn't break strings automatically](https://github.com/google/yapf/issues/408) :(